de.lucaswerkmeister.corebird.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "de.lucaswerkmeister.corebird",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "corebird",
+  "rename-icon" : "corebird",
+  "copy-icon" : true,
+  "finish-args" : [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=home:ro",
+    "--filesystem=~/.corebird/:rw",
```

Comment:
You don't need direct access to this directory.

Reply:
I’m not sure what you mean, sorry. Do you mean that the app doesn’t need access to this directory? That it *shouldn’t* need access to the directory? That I don’t need to specify it in the manifest because it’s included by default?

Reply:
The existing CoreBird Flatpak never had write access here so it was clearly optional for legacy configs.

Reply:
Hm, I might have misunderstood the `--filesystem=host` in the old manifest – does that not include `~/.corebird`?

---

In any event, though – it’s looking like the rename will not preserve backwards compatibility (no objections to that so far in lucaswerkmeister/corebird#5), and in that case this can be dropped anyways.